addons/hr_holidays/models/hr_leave.py

```diff
@@ -959,7 +959,7 @@ def write(self, values):
 
     @api.ondelete(at_uninstall=False)
     def _unlink_if_correct_states(self):
-        error_message = _('You cannot delete a time off which is in %s state')
+        error_message = _("You can't delete an %s time off")
```

Comment:
You can't delete an Second Approval time off

`an` is used only when the following word starts with a vowel sound

Reply:
New version sounds weird.. 

I prefer old better.. considering the state names